ruby_connectors/lib/ruby_connectors/temporal/workflows/read_database_data_workflow.rb

```
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+module RubyConnectors
+  module Temporal
+    module Workflows
+      class ReadDatabaseDataWorkflow < ::Temporal::Workflow
+        DEFAULT_BATCH_SIZE = 10_000
+
+        def execute(params)
+          initialize_workflow_state(params)
+          
```

RuboCop (1.69.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)